.github/workflows/ci.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         run: |
           cd examples/rust_with_cffi/
           python --version
-          pip install -U pip wheel setuptools
```

Comment:
```
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-uninstall-d0uy_7qf\\pip.exe'
Consider using the `--user` option or check the permissions.
```

Upgrade pip on Windows fails with permission error.

Reply:
I think you can get around this by using `python -m pip` instead of `pip` as the executable.

Reply:
Well, I think it's not needed to be upgraded now.